perf/gh-7089-vclock-copy/Makefile

```diff
@@ -0,0 +1,21 @@
+test: test_none stop
+
+test_wal: test_write stop
+
+dirs:
+	for i in {1..11}; do mkdir $${i} || true; done
```

Comment:
Add `SHELL := /bin/bash` at top, because `{1..11}` is the bash specific syntax. It'll not work on dash, which is default non-interactive interpreter on Debian/Ubuntu.

Reply:
Nit: I think that *simply expanded* (`:=` instead of `=`) variable should be preferred for the simplicity sake where possible.

https://www.gnu.org/software/make/manual/html_node/Flavors.html

Reply:
Wow, live and learn. Fixed.